doc/2/api/controllers/document/update/index.md

```diff
@@ -17,7 +17,7 @@ Updates a document content.
 ### HTTP
 
 ```http
-URL: http://kuzzle:7512/<index>/<collection>/<documentId>/_update[?refresh=wait_for][&retryOnConflict=<int>]
+URL: http://kuzzle:7512/<index>/<collection>/<documentId>/_update[?refresh=wait_for][&retryOnConflict=<int>][&source=<boolean>]
```

Comment:
Since it's a boolean flag, if it's in the query string then the flag is set 
```suggestion
URL: http://kuzzle:7512/<index>/<collection>/<documentId>/_update[?refresh=wait_for][&retryOnConflict=<int>][&source]
```